tests/test_file_mapping_models.py

```
@@ -0,0 +1,195 @@
+    FileMappingConfig.model_validate(
+      {"mappings": [mapping_entry(**{field_name: path_value})]}
+    )
+
+  assert_validation_error_location(exc_info.value, ("mappings", 0, field_name))
+
+
+@pytest.mark.parametrize("field_name", ["path", "windows_path", "remote_path"])
+@pytest.mark.parametrize(
+  "path_value",
+  [
+    "/tmp/config.json",
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/config.json"

(S108)